sdkproject/Assets/Mapbox/Unity/Map/AbstractMap.cs

```diff
@@ -126,7 +131,18 @@ protected virtual void Start()
 
 		void TileProvider_OnTileAdded(UnwrappedTileId tileId)
 		{
-			_mapVisualizer.LoadTile(tileId);
+			var utile = _mapVisualizer.LoadTile(tileId);
+			if (!_centerTileSet)
```

Comment:
I agree that's quite bad but couldn't think of anything better (without too much change) to detect the center of the map (first tile). I'll think about this again tomorrow though, it has been a while since I did this.